Objects/codeobject.c

```diff
@@ -1212,8 +1212,8 @@ code.__new__ as code_new
     name: unicode
     firstlineno: int
     linetable: object(subclass_of="&PyBytes_Type")
-    endlinetable: object(subclass_of="&PyBytes_Type")
-    columntable: object(subclass_of="&PyBytes_Type")
+    endlinetable: object
```

Comment:
Looks like we can't use `accept={NoneType}` with `object()` in argument clinic. Should we just do this manual checking or add this ability to argument clinic?

Reply:
I'd go with manual for now, it is not really related to this PR.

Reply:
Sounds good.